common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -204,6 +204,6 @@ def index_dictionary(self):
             xblock_body["content"].update(html_body)
         else:
             xblock_body["content"] = html_body
-        xblock_body["content_type"] = self.category.title()
+        xblock_body["content_type"] = "sequence"
```

Comment:
This will show up in search result display (I think), so we might need to have it start with a capital letter.